casper/src/test/scala/coop/rchain/casper/helper/RhoSpec.scala

```diff
@@ -17,7 +17,7 @@ object RhoSpec {
 
   private def mkRuntime(testResultCollector: TestResultCollector[Task]) = {
     val testResultCollectorService =
-      Seq((4, "assertAck", 24), (1, "testSuiteCompleted", 25))
+      Seq((5, "assertAck", 24), (1, "testSuiteCompleted", 25))
```

Comment:
Why not add the `BodyRef` to the `BodyRefs` object in `Runtime.scala`?

Reply:
Created a ticket for this: https://rchain.atlassian.net/browse/RHOL-1139

Reply:
I will address this in a separate PR